src/test/java/com/vinylteam/vinyl/util/DatabasePreparerForITests.java

```diff
@@ -145,8 +145,9 @@ public void insertUsers(List<User> users) throws SQLException {
                 insertUsers.setString(2, user.getPassword());
                 insertUsers.setString(3, user.getSalt());
                 insertUsers.setInt(4, user.getIterations());
-                insertUsers.setString(5, user.getRole().toString());
-                insertUsers.setBoolean(6, user.getStatus());
+                insertUsers.setBoolean(5, user.getStatus());
+                insertUsers.setString(6, user.getRole().toString());
+                insertUsers.setString(7, user.getDiscogsUserName());
                 insertUsers.addBatch();
             }
             insertUsers.executeBatch();
```

Comment:
I need to set connection.setAutoCommit(true) after every commit later - this connection is reused in other places later, and needs to be unchanged after method call.